src/store/index.ts

```diff
@@ -1,11 +1,16 @@
 import Vue from "vue";
 import Vuex from "vuex";
 
+import Restaurants from "./module/restaurants";
+import Chefs from "./module/chefs";
+import Dishes from "./module/dishes";
+
 Vue.use(Vuex);
 
 export default new Vuex.Store({
-  state: {},
-  mutations: {},
-  actions: {},
-  modules: {}
+  modules: {
+    Restaurants,
+    Chefs,
+    Dishes
+  }
```

Comment:
There is a really nice library that is an alternative to vuex-class, which I use. I find the syntax more appealing - https://github.com/championswimmer/vuex-module-decorators. 

Take a look if you are interested.